### ISSUE TITLE
101: Types people see when they use ":t" is confusing to them

### ISSUE THREAD

**Issue by illia-bobyr** [google/haskell-trainings#12]
tl;dr: Mention that `:t` output will not be useful until 102 material.

This is something we have discussed a few times in the past.
Until typeclasses in 102, types even for simple functions such as `(+)` and `div` are not really that helpful.

We have tried looking into monomorphising ghci output and considered writing a replacement for a limited number of operations in Prelude, so that during 101 people would not see typeclasess.

While been technically challenging, it might also not be the best solution from the educational standpoint.

An alternative is to mention, in 101, that types people will see when they type `:t` involve abstractions covered in 102, and just suggest they wait until 102, to understand them.
We literally start 102 with slides that explain what those types are.
